src/libexpr/include/nix/expr/primops.hh

```diff
@@ -27,7 +27,12 @@ constexpr size_t conservativeStackReservation = 16;
 struct RegisterPrimOp
 {
     typedef std::vector<PrimOp> PrimOps;
-    static PrimOps * primOps;
+
+    static PrimOps & primOps()
```

Comment:
We maybe should test if builtins are still usuable from our c-api after this... @roberth

Reply:
Unfortunately we don't have a test for `nix_register_primop`, but I can see that it's basically just another `RegisterPrimOp::RegisterPrimOp` constructor call site, so we should be in the clear.

Reply:
According to @xokdvium the difference was having this in the header file vs a compilation unit on macOS, which @edolstra also figured out.